modules/spyndex_utils.py

```diff
@@ -16,6 +16,20 @@
     "S2": "swir22",
 }
 
+LANDSAT_BAND_MAPPING = {
```

Comment:
I think we don't need this dictionary. We can use a generic dictionary (e.g. BAND_MAPPING) where we can add all the bands of the platforms we will use.